Tests/Acceptance/ClearRevokedTokensCommandTest.php

```diff
@@ -0,0 +1,193 @@
+
+        $this->assertNull(
+            $this->client->getContainer()->get(AccessTokenManagerInterface::class)->find(
+                FixtureFactory::FIXTURE_ACCESS_TOKEN_REVOKED
+            )
+        );
+        $this->assertNull(
+            $this->client->getContainer()->get(RefreshTokenManagerInterface::class)->find(
+                FixtureFactory::FIXTURE_REFRESH_TOKEN_REVOKED
+            )
+        );
+        $this->assertNull(
+            $this->client->getContainer()->get(AuthorizationCodeManagerInterface::class)->find(
+                FixtureFactory::FIXTURE_AUTH_CODE_REVOKED
+            )
+        );
+    }
+
+    public function testClearRevokedAccessTokens(): void
+    {
+        $output = $this->executeCommand([
+            '--access-tokens' => true,
+        ]);
+
+        $this->assertStringContainsString('Access tokens deleted: 1.', $output);
+        $this->assertStringNotContainsString('Refresh tokens deleted', $output);
+        $this->assertStringNotContainsString('Auth codes deleted', $output);
+
+        $this->assertNull(
+            $this->client->getContainer()->get(AccessTokenManagerInterface::class)->find(
+                FixtureFactory::FIXTURE_ACCESS_TOKEN_REVOKED
+            )
+        );
+        $this->assertInstanceOf(
+            RefreshToken::class,
+            $this->client->getContainer()->get(RefreshTokenManagerInterface::class)->find(
+                FixtureFactory::FIXTURE_REFRESH_TOKEN_REVOKED
+            )
+        );
+        $this->assertInstanceOf(
+            AuthorizationCode::class,
+            $this->client->getContainer()->get(AuthorizationCodeManagerInterface::class)->find(
+                FixtureFactory::FIXTURE_AUTH_CODE_REVOKED
+            )
+        );
+    }
+
+    public function testClearExpiredRefreshTokens(): void
+    {
+        $output = $this->executeCommand([
+            '--refresh-tokens' => true,
+        ]);
+
+        $this->assertStringNotContainsString('Access tokens deleted', $output);
+        $this->assertStringContainsString('Refresh tokens deleted: 1.', $output);
+        $this->assertStringNotContainsString('Auth codes deleted', $output);
+
+        $this->assertInstanceOf(
+            AccessToken::class,
+            $this->client->getContainer()->get(AccessTokenManagerInterface::class)->find(
+                FixtureFactory::FIXTURE_ACCESS_TOKEN_REVOKED
+            )
+        );
+        $this->assertNull(
+            $this->client->getContainer()->get(RefreshTokenManagerInterface::class)->find(
+                FixtureFactory::FIXTURE_REFRESH_TOKEN_REVOKED
+            )
+        );
+        $this->assertInstanceOf(
+            AuthorizationCode::class,
+            $this->client->getContainer()->get(AuthorizationCodeManagerInterface::class)->find(
+                FixtureFactory::FIXTURE_AUTH_CODE_REVOKED
+            )
+        );
+    }
+
+    public function testClearExpiredAuthCodes(): void
```

Comment:
@franjo-zadelj-trikoder 
```suggestion
    public function testClearRevokedAuthCodes(): void
```